src/BenchmarksApps/TechEmpower/Mvc/Database/Db.cs

```diff
@@ -0,0 +1,57 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Internal;
+using Microsoft.EntityFrameworkCore.Storage;
+using Mvc.Models;
+
+#pragma warning disable EF1001 // Using internal EF pooling APIs, can be cleaned up after 6.0.0-preview4
+
+namespace Mvc.Database;
+
+public class Db
+{
+    private readonly PooledDbContextFactory<ApplicationDbContext> _dbContextFactory;
+
+    public Db(AppSettings? appSettings)
```

Comment:
Why is this nullable?

Reply:
it's coming from `Configuration.Get<AppSettings>()` so it's nullable. Either this accepts a nullable one, or we handle that differently if that's undesirable to have nullable in the ctor.

Reply:
Do this instead:

```C#
var appSettings = new AppSettings();
Configuration.Bind(appSettings);
```